convert2rhel/main.py

```diff
@@ -148,6 +148,10 @@ def main_locked():
 
     except _AnalyzeExit:
         breadcrumbs.breadcrumbs.finish_collection(success=True)
+        # Update RHSM custom facts only when thi returns False. Otherwise,
+        # sub-man get uninstalled and the data is removed from the server.
```

Comment:
```suggestion
        # Update RHSM custom facts only when this returns False. Otherwise,
        # sub-man gets uninstalled and the data is removed from the server.
```